src/pages/academy/components/utility.js

```diff
@@ -0,0 +1 @@
+export const url = typeof window !== 'undefined' ? window.location.href : ''
```

Comment:
```suggestion
export const url = typeof window !== 'undefined' ? window.location.href : ''
```
You can use `isBrowser()` for the first part of the check here.